tests/cockpit-tasks-centosci.yaml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: v1
+kind: List
+items:
+- kind: ReplicationController
+  metadata:
+    name: cockpit-tasks
+  spec:
+    replicas: 1
+    selector:
+      infra: cockpit-tasks
+    template:
+      metadata:
+        labels:
+          infra: cockpit-tasks
+      spec:
+        containers:
+        - name: cockpit-tasks
+          image: docker.io/cockpit/tests
+          securityContext:
+            runAsUser: 1111
+            hostNetwork: false
+          env:
+          - name: TEST_JOBS
+            value: '8'
+          - name: TEST_PUBLISH
+            value: sink
+          volumeMounts:
+          - name: secrets
+            mountPath: "/secrets"
+            readOnly: true
+          - name: cache
+            mountPath: "/cache"
+            readOnly: false
+          - name: images
```

Comment:
Wrongly indented, this starts a new  "container:" definition. Please let's not do this here. The /cache/images volume is already shared between all pods, there is no need to deploy umpteen nginx services that all serve from the same directory. We already have an image server in images/cockpit-images.yaml. (This doesn't auto-respawn, but that's issue #196 - let's fix that separately).

So please drop this container.

Reply:
Ok, Should I drop it on bos too?

Reply:
No, let's keep that for now on BOS, this works differently as it uses host paths, not a shared volume. We might make this more efficient in the future, but I'd rather not touch it without thinking hard about it :)